staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -2205,7 +2204,7 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 			enabledFeatures: []featuregate.Feature{features.CustomResourceDefaulting},
 		},
 		{
-			name: "metadata defaults",
+			name: "old test for metadata defaults",
```

Comment:
nit: "v1.15-era test..."? old will not be helpful in about three months

Reply:
and a TODO: remove in follow-up.

Reply:
done